js/bitbay.js

```diff
@@ -933,10 +931,24 @@ module.exports = class bitbay extends Exchange {
     }
 
     async cancelOrder (id, symbol = undefined, params = {}) {
+        if (!params['side']) {
+            throw new ExchangeError (this.id + ' parameter named side is required');
+        }
+        if (!params['price']) {
```

Comment:
Why would they require it... :\

Reply:
I know.  I tested without it and it doesn't throw an error.

Reply:
If it's too annoying for people, we can always look up the price and amount with another API query.

Reply:
@npomfret nope, no worries, it's better to have it required explicitly than to send another implicit request. I'll just add a few minor edits for Python. Thx! )